tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -492,6 +492,22 @@ where
         }
     }
 
+    /// Sets whether or not the length of the [name] of the current thread is
+    /// tracked for padding.
+    ///
+    /// [name]: std::thread#naming-threads
+    pub fn with_thread_names_length_tracking(
```

Comment:
take it or leave it: this method name is a quite a mouthful. what do you think about something like `with_thread_name_padding`, since what this really controls is whether or not the name of the thread is padded when it's printed.